packages/app-shell/src/index.tsx

```diff
@@ -35,10 +33,28 @@ export class AppShell extends React.Component<Props, State> {
   };
 
   state = {
+    breadcrumbs: [],
     data: {},
     toasts: [],
   };
 
+  addBreadcrumb = (label: string, props: Breadcrumb['props'] = {}) => {
+    const id = uuid();
+
+    this.setState(prevState => ({
+      breadcrumbs: [
+        ...prevState.breadcrumbs.filter(crumb => crumb.id !== id),
```

Comment:
is this filter necessary? since `id` is created above for every `addBreadcrumb`?

Reply:
let's remove it and add it when/if we need it for add